test/kill.js

```diff
@@ -262,3 +262,48 @@ test('calling cancel method on a process which has been killed does not make err
 	const {isCanceled} = await t.throwsAsync(subprocess);
 	t.false(isCanceled);
 });
+
+test('calling abort throws an error with message "Command was canceled"', async t => {
```

Comment:
Those tests fail on older Node.js versions due to `AbortController` not being supported. They should be put inside a condition, maybe something like this?

```js
if (globalThis.AbortController !== undefined) {
```